### ISSUE TITLE
HIghlight escaped character in string literal

### ISSUE THREAD

**Issue by pocke** [pocke/iro#28]

**Comment by thinca** [comment]
And highlight meta string in regexp literal is also useful.
But we may need regexp parser if do this in strict.

**Comment by pocke** [comment]
Definitely. But I guess highlighting for regexp is more difficult than String's one.
I think regular expression is enough to highlight escaped characters in **String**, but probably we need a parser (maybe Onigumo) for special character in **regexp** as you said.
So I'll implement this feature for String firstly, then I'll investigate Onigumo.
